tests/components/todoist/test_todo.py

```diff
@@ -402,8 +402,50 @@ async def test_update_todo_item_status(
                 "status": "needs_action",
             },
         ),
+        (
+            [
+                make_api_task(
+                    id="task-id-1",
+                    content="Soda",
+                    description="6-pack",
+                    is_completed=False,
+                    due=Due(date="2024-01-01", is_recurring=True, string="every day"),
```

Comment:
Important part of this test is creating a mock task that has a string value in it's Due object.

Reply:
I realze these tests are super unreadable with so many parameters.... Perhaps it's worth adding your PR comments as code comments?

Reply:
e.g.
```suggestion
                    # Create a mock task with a string value in the Due object and verify it
                    # gets preserved when verifying the kwargs to update below
                    due=Due(date="2024-01-01", is_recurring=True, string="every day"),
```